README.adoc

```diff
@@ -12,7 +12,7 @@
 :page-releasedate: 2019-12-12
 :page-essential: true
 :page-essential-order: 1
-:page-description: Learn how to access and persist data with MongoDB using Contexts and Dependency Injection (CDI) and MicroProfile Config.
+:page-description: Learn how to access and persist data with MongoDB using Contexts, Dependency Injection (CDI) and MicroProfile Config.
```

Comment:
I think use comma is not right. "Contexts and Dependency Injection (CDI)" is one thing. See https://openliberty.io/docs/20.0.0.9/cdi-beans.html